ROADMAP.md

```diff
@@ -0,0 +1,84 @@
+Roadmap
+=======
+
+This document spans the Roadmap for the in-toto main implementation for the
+time window from April 2019 to April 2020. The main theme of this year's
+efforts are focused on two fronts strengthening our release process and aiming
+for version 1.0!
+
+## Strengthening the release process
+
+As a software supply chain security product, in-toto aims to be at the
+forefront of the best practices when it comes to developing the tool. As such,
+the in-toto has the following goals for this year
+
+### CII Silver Badge
+
+in-toto is already marked as "passing criteria" under the Core Infrastructure
+Initiative best practices badge. This year we aim to upgrade our rating to
+Silver. To work towards this goal, we'll review and comply with the few
+missing requirements for the silver badge.
+
+### Thorough Documentation of Release process with in-toto metadata
```

Comment:
Is the mix of capitalization and lower-case intended? The rest of the document has only the first word of the section headings capitalized (except for CII Silver Badge).